src/passes/bytesConverter/replaceIndexAccessBytesConverter.ts

```diff
@@ -0,0 +1,72 @@
+import { FixedBytesType, generalizeType, TypeName, IndexAccess, Literal } from 'solc-typed-ast';
+import { AST } from '../../ast/ast';
+import { ASTMapper } from '../../ast/mapper';
+import { createCallToFunction } from '../../utils/functionGeneration';
+import { generateExpressionTypeString } from '../../utils/getTypeString';
+import { typeNameFromTypeNode } from '../../utils/utils';
+import {
+  createNumberLiteral,
+  createUint8TypeName,
+  createUint256TypeName,
+} from '../../utils/nodeTemplates';
+import { safeGetNodeType } from '../../utils/nodeTypeProcessing';
+import { replaceBytesType } from './utils';
+
+export class ReplaceIndexAccessBytesConverter extends ASTMapper {
+  visitIndexAccess(node: IndexAccess, ast: AST): void {
+    const baseExprType = generalizeType(safeGetNodeType(node.vBaseExpression, ast.inference))[0];
```

Comment:
safeGetNodeType is used multiple times in the method
```suggestion
    const typeNode = safeGetNodeType(node.vBaseExpression, ast.inference);
    const baseExprType = generalizeType(typeNode)[0];
```

Reply:
It was used 2 times with the same node, other calls were with different nodes. Changes Incoming